docs/reference/dependencies.md

```diff
@@ -919,24 +919,24 @@ All of these libraries are required for core functionality.
 - [classnames@2.3.2](https://github.com/JedWatson/classnames) (MIT)
 - [clsx@1.1.1](https://lukeed.com) (MIT)
 - [comma-separated-tokens@2.0.2](https://wooorm.com) (MIT)
-- [component-event@0.1.4](https://github.com/component/event) (MIT\*)
-- [component-props@1.1.1](https://github.com/component/props) (MIT\*)
+- [component-event@0.1.4](https://github.com/component/event) (MIT-)
+- [component-props@1.1.1](https://github.com/component/props) (MIT-)
```

Comment:
Are these - additions intentional?

Reply:
@cigdemilhan nudging this, did you mean to add these dashes?

Reply:
@akeller I did not mean to add these dashes, probably it happened when I did changes the wildcards with dash

Reply:
I'll clean this up before merging 🙂

Reply:
@akeller thanks :)